lib/screen/wallet_connect/send/wc_send_transaction_page.dart

```diff
@@ -49,14 +49,14 @@ class _WCSendTransactionPageState extends State<WCSendTransactionPage> {
   void initState() {
     super.initState();
 
-    final to = EthereumAddress.fromHex(widget.args.transaction.to);
+    final to = EthereumAddress.fromHex(widget.args.transaction.to ?? "");
```

Comment:
better to not go to this screen when the `widget.args.transaction.to` is not valid. since the `EthereumAddress.fromHex` will throw an error if address is invalid then this screen won't load.